### DISCUSSION TITLE
Scheduled charge doesn't start

### DISCUSSION THREAD

**Discussion by ps121988** [lbbrhzn/ocpp#864] [Q&A]
Hello
I've integrated my wallbox viaris orbis uni in home assistant.
After integration , when i schedule charge from producer app , charge doesn't start 
 until i manually switch on Wall box button from ha .
Why ? Any solution or workaround ?

Many thanks

**Comment by drc38** [comment]
Using two backends to control a charger may cause conflicts. Why don't you use HA to schedule the charge via an automation?

**Reply by ps121988** [reply]
Thanks for your answer .
The wall box is shared by different user and some of them don't use HA.
